src/main/util/file_util.py

```diff
@@ -55,3 +62,13 @@ def get_content_from_file(file: str, encoding: str = ISO_ENCODING, to_strip_nl:
     with open(file, 'r', encoding=encoding) as f:
         content = f.read()
         return content if not to_strip_nl else content.rstrip('\n')
+
+
+def pair_in_and_out_files(in_files: list, out_files: list) -> List[Tuple[str, str]]:
```

Comment:
We are expecting that all out_* files have corresponding in_* files in some case_* directory, don't we? If so, i think it would be better to check if `len(in_files) == len(out_files)` inside that function and throw an error if their lengths don't equal.
This way we will be sure that, first of all, every in_* file has corresponding out_* file and, secondly, that all out_* files in directory are involved.  
As for me, it should be the part of the function logic, what do you think?

Reply:
Great idea, thanks! I have added this condition to the `pair_in_and_out_files` function and removed it here